std/node/tests/node_modules/left-pad/index.js

```diff
@@ -16,7 +16,7 @@ var cache = [
   "      ",
   "       ",
   "        ",
-  "         "
+  "         ",
```

Comment:
unrelated change -  please undo

Reply:
I just auto-formatted the whole `std/node` directory by accident. Reverted!